server/src/test/scala/za/co/absa/atum/server/api/repository/FlowRepositoryIntegrationTests.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.repository
+
+import org.junit.runner.RunWith
+import org.mockito.Mockito.{mock, when}
+import za.co.absa.atum.server.api.TestData
+import za.co.absa.atum.server.api.database.flows.functions.GetFlowCheckpoints
+import za.co.absa.atum.server.api.exception.DatabaseError
+import zio._
+import zio.test.Assertion.failsWithA
+import zio.test._
+import zio.test.junit.ZTestJUnitRunner
+
+@RunWith(classOf[ZTestJUnitRunner])
```

Comment:
Remove the annotation and make it an object. Also as above, it's a unit test and should be executed as such. Please rename the object to `FlowRepositoryUnitTests`

Reply:
@lsulak 
Maybe you could also rename all other test files where the suffix IntegrationTests for incorrectly used?

Reply:
naturally, I also did it already :)